examples/sklearn/mpg-estimation/cortex.yaml

```diff
@@ -5,6 +5,3 @@
   name: mpg
   predictor:
     path: predictor.py
-    model: s3://cortex-examples/sklearn/mpg-estimation/linreg.joblib
```

Comment:
Why did you decide to embed the model URL in predictor.py instead of passing it in as metadata?